simulator/index.md

```diff
@@ -8,34 +8,26 @@ title: Simulator
 
 # Simulator
 
-![]({{ site.baseurl }}/images/content/simulator/arena.png)
-
 ## Installation
 
 ### Prerequisites
 
-You need to download and install [Webots](https://cyberbotics.com/#download) (the download is around 1.5GB). This is the platform we run our simulation in. Whilst we recommend version "R2021a" of Webots, and will be running competition simulations using it, "R2020b-rev1" should still work.
+You need to download and install [Webots](https://cyberbotics.com/#download) (the download is around 400MB). This is the platform we run our simulation in. We recommend version "R2021b" of Webots.
 
 #### Python Version
 
 You will also need Python installed.
 
 | Platform | Supported Python Version |
 |----------|--------------------------|
-| Windows  | 3.7 - 3.8 (64-bit)       |
-| macOS    | 3.7 - 3.8                |
-| Linux    | 3.7 - 3.8                |
-
-In the competition, Python 3.7 will be used.
+| Windows  | 3.7+ (64-bit)            |
+| macOS    | 3.7+                     |
```

Comment:
Looking at the webots docs, 3.10 isn't supported yet 

https://www.cyberbotics.com/doc/guide/using-python

Reply:
The physical kit runs 3.9, so best not to encourage use of 3.10 features?

We might be able to upgrade to 3.10 later, but it's been a very problematic release.